src/config/locales/en/history.js

```diff
@@ -453,12 +453,6 @@ export const history = {
         additionalActivity: 'Additional periods of activity with this employer',
       },
       para: {
-        done:
```

Comment:
Were these (para.done, para.exiting, para.employment) unused?

Reply:
Yup, I did a global search for them and they are not being used. These were `history.employment.default.para...`, not to be confused with `history.employment.para...` 🤦‍♀️

Reply:
I can't find any references either, looks like this is fine. There are a couple of places that the keys for i18n are dynamically assembled but these don't seem to be any of those.